packages/components/psammead-inline-link/src/index.stories.jsx

```diff
@@ -1,12 +1,43 @@
 import React from 'react';
+import styled from 'styled-components';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { getBodyCopy, getLongPrimer } from '@bbc/gel-foundations/typography';
+import {
+  getSansRegular,
+  getSansBold,
+  getSansRegularItalic,
+  getSansBoldItalic,
+} from '@bbc/psammead-styles/font-styles';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
 import notes from '../README.md';
 import InlineLink from './index';
 
+const RegularParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansRegular(service)}
+`;
+const BoldParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansBold(service)}
+`;
+const ItalicParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansRegularItalic(service)}
+`;
+const BoldItalicParagraph = styled.p`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansBoldItalic(service)}
+`;
+
+const Caption = styled.figcaption`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => getSansRegularItalic(service)}
+`;
+
 storiesOf('Components|InlineLink', module)
-  .addDecorator(withKnobs)
+  .addDecorator(withKnobs())
```

Comment:
Looking at other implementation of this decorator, I think this should remain as just `withKnobs` without being invoked.